test/functorch/test_ops.py

```diff
@@ -690,6 +691,9 @@ def f(inp, *args, **kwargs):
         # The failure occurred for item [0]
         xfail('masked.prod')
     }))
+    @skipOpsIfTorchDynamo('TestOperators', 'test_vjpvjp', vjp_fail.union({
+        skip('index_reduce'),  # double backward  not really supported
+    }))
```

Comment:
Just so I understand what's going on:
- previously test_vjpvjp_index_reduce passed (under eager-mode PyTorch).
- Now, it fails after this PR.
- The question is: "how did `test_vjpvjp_index_reduce` pass before?"

Also, just checking -- I don't see any vmap tests @eellison, were they a problem in a previous version of this PR?

Reply:
(Or is the CI just obscuring test failures?)